recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set name = "typeguard" %}
 {% set version = "4.1.5" %}
+{% set min_python = "3.8" %}
+{% set min_typing_extensions = "4.7.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: typeguard
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/typeguard/typeguard-{{ version }}.tar.gz
   sha256: ea0a113bbc111bcffc90789ebb215625c963411f7096a7e9062d4e4630c155fd
 
 build:
   noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
```

Comment:
this avoids any upstream `build-system` flailing from surprising conda-forge

Reply:
I read that sentence multiple times without understanding. Build isolation is usually a good thing. Is there a good reason why you're disabling it here?

Reply:
`conda-build` is already an build system, building in an isolated environment, but not the `pip` concept of `build-system`. If _that_ build system allowed to do what it is designed to do, it can install _another_, not-managed-by-conda-build virtual envrionment, and then builds with that.

In CI, this is _generally_ overloaded with the environment variable `PIP_NO_BUILD_ISOLATION`, but while locally debugging, I wanted to make that I was getting what i ask for.